dashboard/app/views/maker/setup.html.haml

```diff
@@ -29,6 +29,10 @@
     border-radius: 50%;
   }
 
+  .validation-children div > p {
```

Comment:
(commenting here because i can't comment on the file in general) ahhh, good find in spotting the styles in this HAML template! i disagree with / don't understand why we decided to style the frontend from the backend, but this is clearly the right spot for these styles to match the current convention, and i don't think it's worth refactoring right now